apps/web/components/getting-started/steps-views/ConnectedVideoStep.tsx

```diff
@@ -36,9 +42,14 @@ const ConnectedVideoStep = (props: ConnectedAppStepProps) => {
                     <AppConnectionItem
                       type={item.type}
                       title={item.name}
+                      isDefault={item.slug === defaultConferencingApp}
                       description={item.description}
                       logo={item.logo}
+                      slug={item.slug}
                       installed={item.userCredentialIds.length > 0}
+                      defaultInstall={
+                        !defaultConferencingApp && item.appData?.location?.linkType === "dynamic"
```

Comment:
We should allow making a conferencing app as default only if there is no existing default conferencing app.